ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Clang/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+##############
+# LLVM/Clang #
+##############
+# Clang is an OpenJ9 prerequisite.
+
+- name: Test if Clang is already installed (required by OpenJ9)
+  win_stat:
+    path: 'C:\Program Files\LLVM\bin\clang.exe'
+  register: clang_installed
+  tags: clang
+
+- name: Download Clang
+  win_get_url:
+    url: http://releases.llvm.org/6.0.1/LLVM-6.0.1-win64.exe
```

Comment:
Please change this to use the https url.

Reply:
Already fixed.